README.md

```diff
@@ -143,34 +145,32 @@ Nothing.
 const MAX_WAKE_TIME = 30;
 const SLEEP_TIME    = 60;
 
-function readTemp(done) {
-    // Take an async temp reading
-    th.setMode(HTS221_MODE.ONE_SHOT);
-    th.read(function(res) {
-        if ("error" in res) {
+function readTemp() {
+    // Take an asynchronous sensor reading
+    sensor.setMode(HTS221_MODE.ONE_SHOT);
+    sensor.read(function(reading) {
+        if ("error" in reading) {
             // Log error
-            server.error("Temperature/Humidity reading error: " + res.error);
+            server.error("Temperature/Humidity reading error: " + reading.error);
         } else {
             // Log reading
-            server.log("Temperature: " + res.temperature + "°C Humidity: " + res.humidity + "%");
-            // Go to sleep
-            done();
+            server.log("Temperature: " + reading.temperature + "°C Humidity: " + reading.humidity + "%");
```

Comment:
This example should call done() parameter.

Reply:
There was a mistake. doAsyncAndSleep action function takes a parameter, a function that should be triggered when the async action is completed. I've updated the docs and the example.